woocommerce/woocommerce
# Cleanup assets in build script ZIP file output.

### Changes proposed in this Pull Request:

- Update the Plugin URI in the main plugin file headers.
- Add a README.md file (included in ZIP file).
- Add a DEVELOPMENT.md file (not included in ZIP file).
- Generate and include a CHANGELOG.md file.
- Ignore some dev-specific files when building ZIP file.
- Ignore the /vendor/ directory when building ZIP file.
- Ignore the /woo-ai/ directory when building ZIP file.

Closes #38446.

### How to test the changes in this Pull Request:

1. Run `pnpm run build:zip`
2. Uncompress the ZIP file.
3. Test that it works properly via upload to an Atomic/self-hosted site.
4. Check that we aren't including any extra files.
